### ISSUE TITLE
Can't load Kernel binary (Dart SDK dev.69.4)

### ISSUE THREAD

**Issue by Marco87Developer** [dart-lang/sdk#34017]
I have a problem after updating the Dart SDK.

* Dart SDK Version: 2.0.0-dev.69.4
* OS: Windows 10
* Browser: Chrome 68

The following is my `pubspec.yaml` file:
```
environment:
  sdk: '>=2.0.0-dev.69.4 <2.0.0'

dependencies:
  sass_builder: ^2.1.1
  angular: ^5.0.0-beta+2
  angular_components: ^0.9.0-beta+2
  angular_forms: ^2.0.0-beta+2
  angular_router: ^2.0.0-alpha+17
  js: ^0.6.1+1
  path: ^1.6.2

dev_dependencies:
  angular_test: ^2.0.0-beta+2
  build_runner: ^0.9.2
  build_test: ^0.10.3+1
  build_web_compilers: ^0.4.1
  test: ^1.3.0
```

I runned `pub global activate webdev` and `pub upgrade` commands. After this, I runned `webdev serve`, but I received the following error message:
```
Can't load Kernel binary: File size is too small to a valid kernel file.
webdev failed with an unexpected exception.
Unable to spawn isolate: Error while initializing Kernel isolate
```

How can I solve this issue? Thanks!

**Comment by cooler-king** [comment]
Also see https://github.com/dart-lang/webdev/issues/70

**Comment by cedx** [comment]
I don't think that the issue is tied to `webdev` : I also have this error when using [Grinder](https://pub.dartlang.org/packages/grinder) with a basic task (i.e. `print('hello')` => `Can't load Kernel binary`).

Windows 10 + Dart SDK 2.0.0-dev.69.4

**Comment by denniskaselow** [comment]
Same problem here. It happens even when I try to start an empty main (or a non existing file) with `dart bin/main.dart`.

The curious thing is, if you use the full path to the dart executable it works just as it is supposed to work. But the output of `dart --version` is exactly the same and I have no other `dart.exe` on my system, so it is the same executable.

Doing `dart --trace-loading bin/main.dart` gives following output:
```
Can't load Kernel binary: File size is too small to a valid kernel file.
* 1bcde Setting working directory: F:\Programming\workspaces\dart\scheduler_updater
* 1bcde Working directory URI: file:///F:/Programming/workspaces/dart/scheduler_updater/
Error while initializing Kernel isolate
```

Using `full\path\to\dart\bin\dart --trace-loading bin/main.dart`:
```
* 9d684 Setting working directory: F:\Programming\workspaces\dart\scheduler_updater
* 9d684 Working directory URI: file:///F:/Programming/workspaces/dart/scheduler_updater/
* 90066 Setting working directory: F:\Programming\workspaces\dart\scheduler_updater
* 90066 Working directory URI: file:///F:/Programming/workspaces/dart/scheduler_updater/
* 90066 Resolving script: bin/main.dart
* 90066 Resolved entry point to: file:///F:/Programming/workspaces/dart/scheduler_updater/bin/main.dart
* 90066 Resolving script: bin/main.dart
* 90066 Resolved entry point to: file:///F:/Programming/workspaces/dart/scheduler_updater/bin/main.dart
```

**Comment by denniskaselow** [comment]
Fixed in the final release of Dart 2.0.0 and Dart 2.1.0-dev.0.0.

**Comment by a-siva** [comment]
Closing issue based on comment above, please reopen if it is still a problem
